Looks like the unit test is failing:

```
--- FAIL: TestHealthCheck (0.01s)
	server_test.go:487: expected status code 500, got 200
	server_test.go:501: expected status code 200, got 500
```

--- a/cluster/saltbase/salt/monit/kubelet
+++ b/cluster/saltbase/salt/monit/kubelet
@@ -3,4 +3,7 @@ group kubelet
 start program = "/etc/init.d/kubelet start"
 stop program = "/etc/init.d/kubelet stop"
 if does not exist then restart
-
+if failed port 10250
+  protocol HTTP request "/healthz"
+  with timeout 10 seconds
+  then restart

--- a/pkg/kubelet/server.go
+++ b/pkg/kubelet/server.go
@@ -165,13 +165,11 @@ func (s *Server) handleHealthz(w http.ResponseWriter, req *http.Request) {
 
 	// Check that the hostname known by the master matches the hostname
 	// the kubelet knows
-	/*
-		hostname := s.host.GetHostname()
-		if masterHostname != hostname {
-			s.error(w, errors.New("Kubelet hostname \""+hostname+"\" does not match the hostname expected by the master \""+masterHostname+"\""))
-			return
-		}
-	*/
+	hostname := s.host.GetHostname()
+	if masterHostname != hostname && masterHostname != "127.0.0.1" && masterHostname != "localhost" {
+		s.error(w, errors.New("Kubelet hostname \""+hostname+"\" does not match the hostname expected by the master \""+masterHostname+"\""))
+		return
+	}
 	w.Write([]byte("ok"))
 }
 

